packages/dashboard/src/index.html

```diff
@@ -7,8 +7,15 @@
 
     <script type="module" src="/build/iot-app-kit-dashboard.esm.js"></script>
     <script nomodule src="/build/iot-app-kit-dashboard.js"></script>
+    <script type="module" src="/build/iot-app-kit-dashboard.esm.js"></script>
```

Comment:
is there a reason we are including a script to the same source twice? (look at ln8)